src/amo/components/CategoryIcon/styles.scss

```diff
@@ -149,3 +153,11 @@ $default-icon-size: 64px;
 .Icon-websites {
   @include icon($name: "websites");
 }
+
+// `experimental` is also the name of a `ui/components/Icon`. Because we use
```

Comment:
Seems like we should just rename the Icon component to avoid the overlap. We could call it...










beta.

Reply:
> beta.

😂